ocw/lib/gce.py

```diff
@@ -207,3 +209,112 @@ def cleanup_all(self):
                 if "warnings" in response:
                     for w in response["warnings"]:
                         self.log_warn(w["message"])
+        self.cleanup_vault_keys()
+
+    def cleanup_vault_keys(self):
+        self.log_info("## GCE Vault key cleaning - %s ##" % self.__project)
```

Comment:
this comment applies to whole PR . please read https://realpython.com/python-string-formatting/ . in project we currently using `#2` . I am considering moving to `#3` some day but for sure against fallback to `#1`

No about this specific message . You are not cleaning any Vault keys here you cleaning service accounts in google cloud . this also refers to function name

Reply:
*this change is ready for review*